decoder.go

```diff
@@ -30,6 +31,8 @@ func Decode(r io.Reader) (*OBJFile, error) {
 
 	scanner := bufio.NewScanner(r)
 	for scanner.Scan() {
+		re, _ := regexp.Compile(`\s+`)
```

Comment:
Would have preferred another way to check that an user expression. 
In any case, the regex compilation should be done outside of the loop, and preferably once for all, with a `regex.MustCompile` global block